packages/loader/driver-utils/src/buildSnapshotTree.ts

```diff
@@ -17,23 +17,23 @@ import { v4 as uuid } from "uuid";
 function flattenCore(
     path: string,
     treeEntries: ITreeEntry[],
-    blobMap: Map<string, string>,
+    blobMap: Map<string, ArrayBufferLike>,
 ): git.ITreeEntry[] {
     const entries: git.ITreeEntry[] = [];
     for (const treeEntry of treeEntries) {
         const subPath = `${path}${treeEntry.path}`;
 
         if (treeEntry.type === TreeEntry.Blob) {
             const blob = treeEntry.value;
-            const buffer = IsoBuffer.from(blob.contents, blob.encoding);
+            const buffer = stringToBuffer(blob.contents, blob.encoding);
             const id = uuid();
-            blobMap.set(id, buffer.toString("base64"));
+            blobMap.set(id, buffer);
 
             const entry: git.ITreeEntry = {
                 mode: FileMode[treeEntry.mode],
                 path: subPath,
                 sha: id,
-                size: buffer.length,
+                size: IsoBuffer.from(blob.contents, blob.encoding).length,
```

Comment:
I'd not do it and write out zero here - it's not used and is not exposed from public function, so no need to waste resources of allocating memory, decoding, etc. - it's a waste that has no value